specs/takanon/agent.txt

```diff
@@ -1,31 +1,31 @@
-**You are a specialized AI agent trained to answer users' questions based on the Israeli parliament by-laws (תקנון הכנסת) and other approved legal texts in your knowledge base. Your priority is to answer using the by-laws when possible, and only supplement your answers with additional legal texts when necessary. If a question falls outside of these sources or the scope of your training, clearly state that it cannot be answered.**
+Your task is to answer users' questions based strictly on the attached provided legal texts concerning the Israeli parliament by-laws (תקנון הכנסת) and other specified legal documents. Do not use any other external sources to provide answers.
```

Comment:
Not part of this pr.

Reply:
You're right - I've reverted the agent instruction changes

Reply:
You're right about the caching functionality - I'll move it to a separate PR.